### ISSUE TITLE
[HIFIVE] cctest/test-assembler-riscv64/RISCV_UTEST_swlwu failed

### ISSUE THREAD

**Issue by luyahan** [v8-riscv/v8#210]
**Describe the bug**

**To Reproduce**
1. Build with options `riscv64.native.debug`
2. Run test/program `riscv64.native.release/cctest test-assembler-riscv64/RISCV_UTEST_swlwu --random-seed=-1998993725 --nohard-abort --enable-slow-asserts --verify-heap --testing-d8-test-runner`
3. See error
```
=== cctest/test-assembler-riscv64/RISCV_UTEST_swlwu ===
#
# Fatal error in ../../test/cctest/test-assembler-riscv64.cc, line 134
# Check failed: converted_res == expected_res (2238380180 vs. 2238380180).
#
#
#
#FailureMessage Object: 0x3ffff8aef8
==== C stack trace ===============================

    /home/yahan/riscv64.native.release/cctest(+0x3461cc8) [0x2abd076cc8]
    /home/yahan/riscv64.native.release/cctest(+0x33b8672) [0x2abcfcd672]
    /home/yahan/riscv64.native.release/cctest(+0x31a1fe2) [0x2abcdb6fe2]
    /home/yahan/riscv64.native.release/cctest(+0xfcb7c8) [0x2ababe07c8]
    /home/yahan/riscv64.native.release/cctest(+0xfd6b10) [0x2ababebb10]
    /home/yahan/riscv64.native.release/cctest(+0x89781e) [0x2aba4ac81e]
    /home/yahan/riscv64.native.release/cctest(+0x81d74c) [0x2aba43274c]
    /lib64/lp64d/libc.so.6(__libc_start_main+0xa6) [0x3fd5819204]
    /home/yahan/riscv64.native.release/cctest(_start+0x2c) [0x2aba4aa228]
Command: riscv64.native.release/cctest test-assembler-riscv64/RISCV_UTEST_swlwu --random-seed=-1998993725 --nohard-abort --enable-slow-asserts --verify-heap --testing-d8-test-runner
--- FAILED ---
```

**Expected behavior**

**Errors/Logs**
If applicable, copy/paste the errors or log messages here.

**Additional context**

**Comment by penguinwu** [comment]
@luyahan Could you check if this one still fails? I suspect it may have been fixed by #228

**Comment by luyahan** [comment]
This test had pass on lastest commit.
